solidity/core/contracts/AbacusConnectionManager.sol

```diff
@@ -85,11 +88,9 @@ contract AbacusConnectionManager is IAbacusConnectionManager, Ownable {
      * @param _inbox the address of the Inbox
      */
     function enrollInbox(uint32 _domain, address _inbox) external onlyOwner {
-        // un-enroll any existing inbox
-        _unenrollInbox(_inbox);
         // add inbox and domain to two-way mapping
         inboxToDomain[_inbox] = _domain;
-        domainToInbox[_domain] = _inbox;
+        domainToInboxes[_domain].push(_inbox);
```

Comment:
I think we'd probably want to avoid duplicates here, so maybe a `require(!isInbox(_inbox))` at the beginning of this function?

Reply:
Great idea